xchange-lakebtc/pom.xml

```diff
@@ -25,6 +25,12 @@
 	<dependencies>
 
 		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.3.2</version>
+		</dependency>
```

Comment:
Fixed it. I've used the same dependency as the one from xchange-btcchina\pom.xml, so I suppose that one has the same problem.